backend/indexing/management/commands/process_scans.py

```diff
@@ -0,0 +1,29 @@
+import logging
+from es.client import elasticsearch
+from es.search import get_index
+from django.core.management import BaseCommand
+
+from addcorpus.python_corpora.load_corpus import load_corpus_definition
+
+log = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    '''Process scan media files based on ES index.
+    See CorpusDefinition.process_scan()'''
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'corpus',
+        )
+
+    def handle(self, corpus, **kwargs):
+        corpus_definition = load_corpus_definition(corpus)
+
+        client = elasticsearch(corpus)
+        index = get_index(corpus)
+        results = client.search(index=index)
```

Comment:
This method will fetch 20 documents by default, which is not what you want; you can provide a `size` but it's capped at 10.000. You can use [scan](https://elasticsearch-py.readthedocs.io/en/v7.15.2/helpers.html#scan) to get all documents in the index.